.circleci/config.yml

```diff
@@ -1279,9 +1279,14 @@ workflows:
         - not: << pipeline.parameters.generate_snapshots >>
         - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
     jobs:
-      - lint
-      - run-test-ios-17
-      - pod-lib-lint
+      - lint:
+          name: "swiftlint"
+      - run-test-ios-17:
+          name: "Unit tests"
+      - pod-lib-lint:
+          name: "Pod lib lint"
+      - spm-revenuecat-ui-ios-17:
+          name: "RevenueCatUI tests"
```

Comment:
let me know if the name feels like it's not adding much and / or making it harder to map the jobs as they show up in github to the circleCI config. I just did this because the name `spm-revenuecat-ui-ios-17` seemed so specific that it didn't seem like a vital check to have until I realized what it was

Reply:
Hmm how about renaming the job to something like `run-revenuecat-ui-ios-17` instead? I feel changing the name like this might make it harder to try to find what job is failing later.

Reply:
done!